pkg/rest/apisurface.go

```diff
@@ -368,6 +368,9 @@ func unpackUnbindRequest(r *http.Request) (*osb.UnbindRequest, error) {
 	vars := mux.Vars(r)
 	osbRequest.InstanceID = vars[osb.VarKeyInstanceID]
 	osbRequest.BindingID = vars[osb.VarKeyBindingID]
+	// plan_id is set in the query string parameters and thus needs to
+	// be obtained differently than instance_id and binding_id.
+	osbRequest.PlanID = r.URL.Query().Get(osb.VarKeyPlanID)
```

Comment:
I personally prefer to use `r.FormValue(osb.VarKeyPlanID)` which I believe is also what has been used previously in this file.

Reply:
Fixed as suggested.